scripts/.storybook/manager-head.html

```diff
@@ -1 +1 @@
-<link rel="icon" type="image/png" href="/logo.png" sizes="32x32" />
+<link rel="icon" type="image/png" href="/favicon.png" sizes="32x32" />
```

Comment:
I don't see this file anywhere, is it a Storybook default?

Reply:
not a default, I renamed `logo.png` in public to `favicon.png` but slipped away during rebase, thanks 😅